LibTest/js_interop/JSFunction/jsFunction_t01.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A JavaScript `Function` value.
+///
+/// @description Check that `JSNumber` can store Dart `int` and `float` values.
+/// @author sgrekhov22@gmail.com
+
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
+import '../js_utils.dart';
+
+main() {
```

Comment:
What is this test testing? That a `JSFunction` can be passed around round-trip? If so, instead of evaluating the function, we could use `===` to make sure that the underlying JS function is still the same as before.